jenkins/revert-master.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -o errexit
+read -p "Delete master [press enter]"
+gcloud compute instances delete pull-jenkins-master
+read -p "Delete master disks [press enter]"
+gcloud compute disks delete pull-jenkins-master pull-jenkins-master-data pull-jenkins-master-docker
+
+read -p "Recreate master disks from snapshot [press enter]"
+gcloud compute disks create pull-jenkins-master --source-snapshot=cadgdhszak0z --size=500GB
```

Comment:
where do these magic snapshot names come from? can we use something more meaningful?